clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -60,7 +60,8 @@ scheduler_tmpl_t<RecordType, ReaderType>::init(
             new scheduler_replay_tmpl_t<RecordType, ReaderType>);
     } else {
         // Non-dynamic and non-replay fixed modes such as analyzer serial and
-        // parallel modes.
+        // parallel modes.  Although serial does do interleaving by timestamp
```

Comment:
Maybe eventually; it is not there yet.  There are other things besides the mapping in the options too.

Expanded the comment to:

    /// XXX i#6831: These interfaces between the main class the subclasses could be
    /// more clearly separated and crystalized.  One goal is to avoid conditionals
    // in scheduler_impl_tmpl_t based on options_mapping or possibly on options_
    // at all (possibly only storing options_ in the subclasses).